x-pack/plugins/lens/public/visualizations/xy/to_expression.ts

```diff
@@ -331,14 +331,15 @@ export const buildXYExpression = (
 
   const layeredXyVisFn = buildExpressionFunction<LayeredXyVisFn>('layeredXyVis', {
     legend: buildExpression([legendConfigFn]).toAst(),
-    fittingFunction: state.fittingFunction || 'None',
-    endValue: state.endValue || 'None',
-    emphasizeFitting: state.emphasizeFitting || false,
-    fillOpacity: state.fillOpacity || 0.3,
-    valueLabels: state?.valueLabels || 'hide',
-    hideEndzones: state?.hideEndzones || false,
-    addTimeMarker: state?.showCurrentTimeMarker || false,
-    valuesInLegend: state?.valuesInLegend || false,
+    fittingFunction: state.fittingFunction ?? 'None',
+    endValue: state.endValue ?? 'None',
+    emphasizeFitting: state.emphasizeFitting ?? false,
+    minBarHeight: state.minBarHeight ?? 1,
```

Comment:
Adding `minBarHeight` here was the only way I could get the expression args to update with the correct value.

Reply:
Yes this is required